modules/ossm-kiali-service-mesh.adoc

```diff
@@ -14,4 +14,4 @@ Installing Kiali  via the Service Mesh on {product-title} differs from community
 * Ingress has been enabled by default.
 * Updates have been made to the Kiali ConfigMap.
 * Updates have been made to the ClusterRole settings for Kiali.
-* Users should not manually edit the ConfigMap or the Kiali custom resource files as those changes might be overwritten by the {ProductShortName} or Kiali operators. All configuration for Kiali running on {ProductName} is done in the `ServiceMeshControlPlane` custom resource file and there are limited configuration options. Updating the operator files should be restricted to those users with cluster-admin privileges.
+* Users should not manually edit the ConfigMap or the Kiali custom resource files as those changes might be overwritten by the {ProductShortName} or Kiali operators. All configuration for Kiali running on {ProductName} is done in the `ServiceMeshControlPlane` custom resource file and there are limited configuration options. Updating the operator files should be restricted to those users with `cluster-admin` privileges. If you are using {product-dedicated}, updating the operator files should be restricted to those users with `dedicated-admin` privileges.
```

Comment:
nit, maybe start with "Do not edit the config map or the Kiali..." "should" seems iffy.

s/ConfigMap/config map/ regardless. Seems like a "general reference": 

https://github.com/openshift/openshift-docs/blob/master/contributing_to_docs/doc_guidelines.adoc#general-references

s/operator/Operator/g (2x)

I'll repeat my sugg for "If you use" instead of "If you are using".